README.md

```diff
@@ -147,9 +147,9 @@ export default Ember.ArrayController.extend({
 
   // binding the property on the paged array
   // to the query params on the controller
-  pageBinding: "content.page",
-  perPageBinding: "content.perPage",
-  totalPagesBinding: "content.totalPages",
+  page: Ember.computed.alias("content.page"),
+  perPage: Ember.computed.alias("content.perPage"),
+  totalPages: Ember.computed.alias("content.totalPages"),
```

Comment:
A few lines below these ones there are duplicated definitions for properties `page` and `perPage`. I believe this particular code example won't work as expected. Either the property definitions below override these ones, or they are ignored.

My question is, how to really address this issue under the new Ember versions that deprecated `-Binding`-like properties?